dcpy/utils/s3.py

```diff
@@ -354,7 +352,7 @@ def get_filenames(bucket: str, prefix: str) -> set[str]:
 def get_subfolders(bucket: str, prefix: str, index=1) -> list[str]:
     prefix = _folderize(prefix)
     prefix_path = Path(prefix)
-    subfolders = set()
+    subfolders: set[str] = set()
```

Comment:
surprised we're not getting a dangerous default here, since sets are mutable 🤷‍♂️